src/modules/nim.rs

```diff
@@ -0,0 +1,57 @@
+use ansi_term::Color;
+use std::process::Command;
+
+use super::{Context, Module};
+
+/// Creates a module with the current Nim version
+///
+/// Will display the Nim version if any of the following criteria are met:
+///     - Current directory contains a `.nim` file
+///     - Current directory contains a `.nimble` file
+pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
```

Comment:
This looks fine to me, but I'd like to have someone else have a look at these files, as language modules haven't really been the area I've worked on in Starship.